pulsar-client-reactive-api/src/main/java/org/apache/pulsar/reactive/client/api/ReactiveMessageSenderBuilder.java

```diff
@@ -173,6 +174,14 @@ default ReactiveMessageSenderBuilder<T> lazyStartPartitionedProducers(boolean la
 		return this;
 	}
 
+	default ReactiveMessageSenderBuilder<T> property(String key, String value) {
```

Comment:
We could also name it `addProperty` ?

Reply:
Please add the property method in a separate PR